src/pageLayout/containers/MainNavigation.tsx

```diff
@@ -417,7 +417,7 @@ export const MainNavigation: FC = () => {
               />
             )}
           />
-          {CLOUD && isContractCustomer && (
+          {CLOUD && (accountType === 'paid' || accountType === 'contract') && (
```

Comment:
Please put this in a variable like:

`const isContractOrPAYG = accountType === 'paid' || accountType === 'contract'`

Then you can check:

`CLOUD && isContractOrPAYG`

We also need a separate case for free accounts, indicating that upgrade is required in relation to the support plan.